src/entities/game/model/store.ts

```diff
@@ -0,0 +1,32 @@
+import { defineStore } from 'pinia';
+import { GAME_STATUS } from './types';
+
+interface State {
+  status: GAME_STATUS;
+  isPaused: boolean;
+  result: any;
+}
+
+export const useGameStore = defineStore('game', {
```

Comment:
OK, but perhaps this will not be necessary.
May be we will just use a hook without store. I can't imagine use-case for global store usage for now.
Pls try to not do things that doesn't really needed for now, because it make noise in code which makes development harder in future.